tests/generic_config_updater/conftest.py

```diff
@@ -135,6 +135,7 @@ def ignore_expected_loganalyzer_exceptions(duthosts, rand_one_dut_hostname, loga
             ".*ERR swss[0-9]*#orchagent.*removeLag.*",  # Valid test_portchannel_interface
             ".*ERR kernel.*Reset adapter.*",  # Valid test_portchannel_interface replace mtu
             ".*ERR swss[0-9]*#orchagent: :- getPortOperSpeed.*",  # Valid test_portchannel_interface replace mtu
+            ".*ERR systemd.*Failed to start Host core file uploader daemon.*",  # Valid test_syslog
```

Comment:
I understand "GCU will reset-failed service if syslog reach 'start-limit-hit' by design." How does it related to validate this error message related to "Host core file uploader daemon"?

Reply:
Offline discussed, the reason is that Host core file uploader daemon depends on rsyslog service.